README.md

```diff
@@ -115,6 +146,15 @@ As Android will not create our main activity before launching the app, we need t
 </resources>
 ```
 
+For wix/react-native-navigation (this is last step, SKIP 4 step!!!)
+
+Add to AppTheme:
+```xml
+        <item name="android:windowBackground">@drawable/bootsplash</item>
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
This will cause jumping status bar issues. Enforcing `windowDrawsSystemBarBackgrounds` is not a solution.

Reply:
With **windowBackground** splash show instantly. **windowDrawsSystemBarBackgrounds** prevents jumping splash screen (between background image and view image). But this works only on 21 api level.

Reply:
I personally use windowDrawsSystemBarBackgrounds: false on API 21+.

Reply:
Without **windowDrawsSystemBarBackgrounds** splash will be jumping( But now, this is the only solution I have found.

Reply:
Yeah, it's because of `android:windowBackground`. This option cannot be used.

Reply:
Without `android:windowBackground` I have a white screen before splash screen.

Reply:
Then the solution is probably too hacky.

Reply:
I need to check `react-native-navigation` more deeply to be sure implementing proper support. But with https://reactnavigation.org/docs/en/next/native-stack-navigator.html coming, I'm not sure the lib is still interesting.

Reply:
Many are still using the Wix library and it appears to be actively maintained.  I appreciate you considering it!

Reply:
Please do not abandon support for react-native-navigation. IMO it's the best nav lib available for a true native experience on device.

Reply:
Just stumbled on this. I also would like this to work on RNN.

Reply:
`react-native-screens` also offers true native navigation experience without the odd setup.

I'm not against offering support for the lib, but I don't want to introduce workarounds and prerequisites only for it.

Currently, this lib:
- Allow you to display the `LaunchScreen.xib` or the `RNBootSplashActivity` at app start.
- Once your app is started, it is hidden, and the splash screen is displayed above the React Native root view (it switches invisibly, the goal here is not to use hacks to extend the splash screen time visibility time).
- When you are ready and call `hide()` from JS, the pseudo splash screen view inside the react root view is hidden.

`react-native-navigation` doesn't only use the default RootView, it replaces it with the navigator and integrates a React root inside each of the screens. It's quite similar to existing app integration: https://facebook.github.io/react-native/docs/integration-with-existing-apps: you only use RN for views in your app (each screen content), not for the whole app. That's why it does not handle this lib correctly.